package.json

```diff
@@ -140,5 +140,8 @@
     "plugins": [
       "preset-webpro"
     ]
+  },
+  "volta": {
+    "node": "18.20.4"
```

Comment:
I'm OK with including this, but would it make sense to use the same range as `engines.node`?

Reply:
Oops, sorry!  I didn't mean to commit that.  `volta` doesn't support ranges.  That was meant purely for my local config.  I will remove.

Reply:
Removed.